### ISSUE TITLE
md-tabs in combination with flex and md-dynamic-height

### ISSUE THREAD

**Issue by reinos** [angular/material#7838]
Currently we have the following structure.

a row with 2 boxes.

![structure](https://cloud.githubusercontent.com/assets/633730/14207814/ae96e75e-f81c-11e5-85da-3dc57dcec7fc.png)

in the second box, there is a tab with different content so also with different heights. What we want is that the tab content is vertical stretching. So we add some flex on the tabs

```
<md-tabs md-selected="tabs.activeTab" md-border-bottom md-autoselect ng-if="dataSet" class="stretch-height" layout="column" flex>
                    <md-tab ng-repeat="tab in tabs"
                            label="{{tab.name}}" layout-fill>
                        <md-tab-content flex layout-fill>
                            <div class="data-app-tab tab{{$index%4}} full-height ng-include" src="tab.url"></div>
                        </md-tab-content>
                    </md-tab>
                </md-tabs>
``` 

This works great when the first/left box is greater then the tab content, as you can see in the screenshot below.

![screen shot 2016-04-01 at 15 09 04](https://cloud.githubusercontent.com/assets/633730/14207896/0563c994-f81d-11e5-837f-98b2cd7d4228.png)

but this works not great when the first/left box is smaller, see the screenshot below.

![screen shot 2016-04-01 at 15 19 37](https://cloud.githubusercontent.com/assets/633730/14207937/2f5f2770-f81d-11e5-8295-271629ce2f5d.png)

So, we though we add the `md-dynamic-height` directive that controls the height and set the height based on the tab. But that is not working, see the screenshot below. The content is now missing.

![screen shot 2016-04-01 at 15 21 21](https://cloud.githubusercontent.com/assets/633730/14207991/6c28d49e-f81d-11e5-8e7e-20d1aba5ecdb.png)

Its there in the dom, but it has an height of zero.

Basically what we want; the structure as seen on the first page, where the tab content is the same as the left/first box when the first/left box is greater. Or when the left/first box is smaller, the first/left box is equal to the tab box and that the tab box is aware of the dynamic height of the boxes.

Ideal:
left/first box is smaller
![screen shot 2016-04-01 at 15 25 20](https://cloud.githubusercontent.com/assets/633730/14208132/1e569e1c-f81e-11e5-942a-49536987c526.png)

Left/first box is greater
![screen shot 2016-04-01 at 15 26 04](https://cloud.githubusercontent.com/assets/633730/14208151/36835bce-f81e-11e5-8826-fe1348d42de0.png)

**Comment by vapits** [comment]
+1

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort. 
      For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).

**Comment by anooppvkeloth** [comment]
i have two tabs on the same page..but the first tab is scrolling properly ..but second tab is not scrolling.
